app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+
+<a href="/is-teaching-right-for-me/computing">You can learn more about teaching computing</a>. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=11&visa_status=false">Search for computing teacher training courses</a>.
+
+## Dance 
+
+As a dance teacher you'll teach pupils performance skills and a range of dance styles and prepare them for professional work. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=12&visa_status=false">Search for dance teacher training courses</a>.
+
+## Design and technology 
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=design_and_technology&button=#funding-widget">Tax-free bursaries of £24,000 are available for eligible trainee design and technology teachers</a>.
+
+As a design and technology teacher you'll encourage pupils to think creatively and design solutions to everyday problems. 
+
+You can <a href="https://www.thenational.academy/teachers/programmes/design-technology-secondary-ks3-l/units">explore what teaching a design and technology lesson could look like</a>.
+
+ <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=DT&visa_status=false">Search for design and technology teacher training courses</a>.
+
+## Drama 
+
+As a drama teacher you'll help pupils develop their acting skills and encourage them to embrace their creativity and imagination. 
+
+You can <a href="https://www.thenational.academy/teachers/programmes/drama-secondary-ks3-l/units">explore what teaching a drama lesson could look like</a>.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=13&visa_status=false">Search for drama teacher training courses</a>.
+
+## Economics 
+
+As an economics teacher you'll help pupils understand the world they live in by learning about the current and historic economic issues at a national and global level. 
+
+ <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=L1&visa_status=false">Search for economics teacher training courses</a>.
+
+## English 
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=english&button=#funding-widget">Tax-free bursaries of £10,000 are available for eligible trainee english teachers</a>. 
```

Comment:
@caitlin-reidy same as previous comments re moving line of bursaries to after intro text.